services/core-web/src/components/Forms/reports/ReportPermitRequirementForm.tsx

```diff
@@ -24,6 +24,7 @@ import RenderGroupCheckbox, {
 } from "@mds/common/components/forms/RenderGroupCheckbox";
 import { getLatestAmendmentByPermitGuid } from "@mds/common/redux/selectors/permitSelectors";
 import RenderRadioButtons from "@mds/common/components/forms/RenderRadioButtons";
+import { maxLength } from "@common/utils/Validate";
```

Comment:
```suggestion
import { maxLength } from "@mds/common/redux/utils/Validate";
```

Reply:
wait a second, just noticed there's already an import from common Validate up on line 16, I change my suggestion to that. 😂